frontend/esbuild.config.js

```diff
@@ -0,0 +1,64 @@
+const { context, build } = require("esbuild");
```

Comment:
mimic what we have for the legacy site (`esbuild.config.js`), but with additional improvement